modules/label_module.rb

```diff
@@ -0,0 +1,24 @@
+require_relative '../label'
+require 'json'
+require 'date'
+
+module LabelsController
+  def load_labels
+    file = './json_files/labels.json'
+    data = []
+    if File.exist?(file) && File.read(file) != ''
+      JSON.parse(File.read(file)).each do |element|
+        data.push(Label.new(id: element['id'].to_i, title: element['title'], color: element['color']))
+      end
```

Comment:
- [ ] Labels are saved in the JSON file but when I tried to fetch, it says "There are no labels yet!". Please fix this.
As you can see in the screenshot below. We have labels stored but when tried to fetch, it says no label.

![Screenshot from 2022-02-25 21-18-59](https://user-images.githubusercontent.com/22434660/155745471-14fa6b7b-575b-4036-9a48-55aeb2f0f930.png)